src/docs/Contributing/Designers/NamingConvention/index.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: 'Naming convention'
+order: 5
+---
+
+> Mozaic is a product used by a lot of people. When so many people work together from different places on such a big project, you need to establish some ground rules that everyone needs to follow. These rules must be followed across every tool your team is using to create consistency and simplify the collaboration process.
+
+We chose to use english in all our files to be understandable by the most. We also chose to use the PascalCase standard in our prototyping tools.
+
+## File Naming Conventions
+
+We use two letters prefixes to name our files, this two letters depending on the pattern type and look like that: **_[XX] File name.sketch_**
```

Comment:
~depending on the pattern type and look like that:~ depends on the pattern and follow the pattern below:

Reply:
sorry guys. I made one mistake. "depends on the type of pattern as you can see below:"